taskfile.js

```diff
@@ -0,0 +1,36 @@
+export async function compile(task) {
+  await task.parallel(['src']);
+}
+
+export async function src(task, opts) {
+  await task
+    .source(
+      opts.src || [
+        'src/**/*.+(js|jsx|ts|tsx)',
+        '!src/**/*.spec.+(js|jsx|ts|tsx)',
+        '!src/**/*.snap',
+      ],
+    )
+    .babel({
+      presets: ['@babel/preset-env', '@babel/preset-react'],
+      plugins: [
+        '@babel/plugin-proposal-object-rest-spread',
+        '@babel/plugin-transform-react-jsx',
+      ],
+    })
+    .target('./dist', { mode: '0755' });
```

Comment:
The output in the `dist` folder is `jsx` for some files 🤔 

Also there are some differences in the output, putting that down to babel@7?

Reply:
🔥️ removed `taskr` went back to `@babel/cli`